dist-material/release-docs/LICENSE

```diff
@@ -398,10 +398,6 @@ EPL licenses
 The following components are provided under the EPL License. See project link for details.
 The text of each license is also included at licenses/LICENSE-[project].txt.
```

Comment:
You should remove these licenses.

Reply:
been removed, forget to update this before.